### PR TITLE
feat: add search bar

### PR DESCRIPTION
**Pull request by will249**
https://trello.com/c/GxNY2O2d/47-2-aauser-i-can-type-my-question-into-a-search-bar

### CODE REVIEW

File: src/app/components/textInput.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+'use client';
```
Comment (matt-newhall) [inline src/app/components/textInput.tsx:1]: `'use client';` ?

File: src/app/components/textInput.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { useState } from 'react';
+
+export const TextInput = () => {
```
Comment (matt-newhall) [inline src/app/components/textInput.tsx:4]: sounds like we may need a `TextInput.test.ts` ;)

File: src/app/components/textInput.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  const onSubmit = () => {
+    console.log('User input:', input);
+  };
+
+  return (
+    <div className="flex flex-row space-x-sm">
+      <input
+        className="block flex-1 rounded-md border border-grey px-sm py-sm placeholder-grey sm:text-sm"
+        type="text"
+        placeholder="Enter your question"
```
Comment (matt-newhall) [inline src/app/components/textInput.tsx:16]: nice!

File: src/app/components/textInput.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  const [input, setInput] = useState('');
+
+  const onSubmit = () => {
+    console.log('User input:', input);
+  };
+
+  return (
+    <div className="flex flex-row space-x-sm">
+      <input
+        className="block flex-1 rounded-md border border-grey px-sm py-sm placeholder-grey sm:text-sm"
```
Comment (matt-newhall) [inline src/app/components/textInput.tsx:14]: px-sm & py-sm -> `p-sm`?

File: tailwind.config.cjs [hunk 5]
```diff
@@ -1,7 +1,38 @@
...
+        green: '#6abd9e',
+        dark_green: '#33a852',
+        yellow: '#fabc05',
+        orange: '#fe6d01',
+        blue: '#17539F',
+        light_blue: '#4284f3',
+        red: '#ea4335',
+        light_grey: '#e1e1e1',
+        x_light_grey: '#dadada',
+        grey: '#666666',
```
Comment (matt-newhall) [inline tailwind.config.cjs:24]: not objecting but assuming these are from cockpit aha?

File: src/app/components/textInput.tsx (hunk 2 above)
Comment (will249) [reply src/app/components/textInput.tsx:4]: oi leave me alone

File: tailwind.config.cjs (hunk 5 above)
Comment (will249) [reply tailwind.config.cjs:24]: haha yeah, until we have designs + theme

**Pull request closed without merging**